backend/routes/user/auth.ts

```diff
@@ -8,7 +8,7 @@ import {
 // import { userLogout } from "../../modules/user/controllers/auth/userLogout.ts";
 // import { getNewAccessToken } from "../../modules/user/controllers/auth/getNewAccessToken.ts";
 // import { authenticateUser } from "../../modules/user/controllers/auth/authenticateUser.ts";
-import { authenticateJWT } from "../middleware/authenticateJWT.ts";
+import { authenticateJWT } from "@/middleware/authenticateJWT.ts";
```

Comment:
ts config has path aliases setup, why are not you using them? instead of having relative imports everywhere